client/src/components/Project/index.jsx

```diff
@@ -14,76 +13,147 @@ class Scrums extends Component {
     },
     scrums: [],
     scrumName: '',
-  }
+    error: {
+      status: false,
+      msg: ''
+    }
+  };
 
   componentDidMount() {
-    // fetch
-    const project = {
-      id: 1,
-      projectName: 'Demo'
-    }
-    this.setState({ scrums, project });
+    // fetch project data
+    const { projectId } = this.props.match.params;
+    axios
+      .get(`/api/v1/projects/${projectId}`)
+      .then(result => {
+        const {
+          data: { data },
+          status
+        } = result;
+        if (status === 200) {
+          const project = {
+            id: data.id,
+            projectName: data['name']
```

Comment:
@AymanAlqoqa You could do `projectName: data.name`?  It's basically the same but then eslint might stop yelling at you :)

Reply:
ok